test/core/image/test_fixture.hpp

```diff
@@ -38,6 +38,19 @@ using image_types = std::tuple
     gil::rgba32_image_t
 >;
 
+using multiband_image_types = std::tuple
```

Comment:
Nitpick: I'd name it `interleaved_image_types` or `rgb_interleaved_image_types`

Reply:
Well multiband is for the opposite of grayscale (single band). Interleaved is rather opposite of planar. `rgb` is a specialization of mutliband. Makes sense?
`gil::gray8_image_t img1(gil::gray8_planar_image_t{}); ` is not implemented (fails to compile) because `struct homogeneous_color_base<Element, Layout, 1>` does not have the full implementation that the other specializations for multibands have. Not sure why. Probably because single band planar image is useless?

Reply:
I changed for `rgb_interleaved_image_types`.

Reply:
> Well multiband is for the opposite of grayscale (single band). Interleaved is rather opposite of planar. rgb is a specialization of mutliband. Makes sense?

Yes, that is how I understand it too.

> Probably because single band planar image is useless?

Conceptually, single band planar should be equivalent to interleaved grayscale,

Details of the issues are collected in #478